### ISSUE TITLE
Swedish letter Å Ä Ö

### ISSUE THREAD

**Issue by ericedberg** [F5OEO/rpitx#330]
Swedish letter Å Ä Ö does not work in Pocsag. Hopefully that could be fixed :)

**Comment by mfalkvidd** [comment]
Pocsag uses 7-bit ascii. There is no ÅÄÖ in 7-bit ascii.

Did minicall have some tweak that added support for ÅÄÖ?

**Comment by ericedberg** [comment]
If you scan and decode 161,4375mhz there will be message with ÅÄÖ.  12 juli 2024 kl. 23:12 skrev Mikael Falkvidd ***@***.***>:﻿
Pocsag uses 7-bit ascii. There is no ÅÄÖ in 7-bit ascii.
Did minicall have some tweak that added support for ÅÄÖ?

**Comment by mfalkvidd** [comment]
Cool. Should be fairly easy to reverse-engineer then. What software do you use for decoding? Multimon-ng?

**Comment by ericedberg** [comment]
https://www.discriminator.nl/pdw/index-en.html

I use PDW and old school police-scanner modified  whit discraminator-tap connected to line-in/mic to a PC.

**Comment by VasylSamoilov** [comment]
The way POCSAG works usually lowercase Engish letters are replaced with other alphabet. You need to build your own codepage. You also may need to use hex codes for some symbols. rpitx accepts and will send anything from 0x00 to 0X7F. Different pagers will have different codepage, there is no standard encoding in pocsag (except English), but when you switch language in pager firmware it no longer applies.

**Comment by mfalkvidd** [comment]
Some info that might be useful in this issue: https://github.com/EliasOenal/multimon-ng/issues/69

**Comment by VasylSamoilov** [comment]
This is correct. Character table (hex code received to character on screen mapping) is different for pager model+pager language combination. 
Example: motorola advisor+Swedish will be different from phillips messanger lux+Swedish, and some pager like motorola alpha bravo will only display English uppercase (cause it's screen is one-line limited character display).
